src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -214,11 +214,56 @@ ProbeVisitor::ProbeVisitor(ASTContext &C, Rewriter &rewriter,
                            set<Decl *> &m, bool track_helpers) :
   C(C), rewriter_(rewriter), m_(m), track_helpers_(track_helpers) {}
 
+bool ProbeVisitor::assignsExtPtr(Expr *E, int *nbAddrOf) {
```

Comment:
I'm not sure this is the best API to retrieve the number of addrof. Should we instead return the number with a special value if the assignment isn't that of an external pointer?

Reply:
I think this interface is okay.